conf/messages.en

```diff
@@ -621,6 +612,24 @@ declaration.itemType.statisticalValue.error.empty = The statistical value cannot
 declaration.itemType.statisticalValue.error.length = Entered statistical value is too long
 declaration.itemType.statisticalValue.error.wrongFormat = Format of this value is incorrect
 
+declaration.commodityDetails.title = Commodity details
```

Comment:
@SteveSugden  what you think about start using `-` in names like `commodity-details` ? It looks very similar like in css and i think is more readable then camel case specially for longer names

Reply:
I've no preference - but this is the style used to-date in this file